backend/src/bundles/auth/auth.controller.ts

```diff
@@ -192,13 +192,6 @@ class AuthController extends ControllerBase {
     ): Promise<ApiHandlerResponse> {
         const [, token] = options.headers.authorization?.split(' ') ?? [];
 
-        if (!token) {
```

Comment:
?

Reply:
That functionality is already released in authorization plugin
![Screenshot_1](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/67747035/e7b9c009-ecb8-4824-ad43-fc93f429111a)